DEVELOPMENT.md

```diff
@@ -47,5 +47,5 @@ No change should be approved until there is documentation for it, supplied in an
 We should always be as transparent as possible.  Sometimes there will be discussions that use customer names, sensitive use cases, and so on.  These must be anonymized, discussed in a private repository, or conducted offline.
 
  - Asynchronous discussions should live in the GitHub issues for this project.
- - Realtime discussions should be in a public chat such as IRC or Slack.
+ - Realtime discussions should be in a public chat such as IRC or the #spec channel on the OpenAPI Slack.
```

Comment:
Thanks @DavidBiesack.  I've added links to Slack.  

I'm not aware of active OAS discussion on IRC and have thus removed it.